pkg_template.html

```diff
@@ -82,10 +80,8 @@
         </p>
         <section class="versions" id="versions">
           <div onclick="load_readme('_version', scroll_to_div=true)" id="_version">
-            <a>
-              _version
-            </a>
             <a href="_link">
+              _version
```

Comment:
This causes that if i click on the text of the version this one doesn't get selected, so the only way to select a version in order to see the README is clicking on the div, there is no way to tell where the div starts or ends

Reply:
Right... Any idea how to fix that ?

Reply:
I just moved the `onclick` from the div to the anchor element. Seems to work !

Reply:
No, that didn't work i can't click neither the div or the anchor, let me search on what to do here i think is possible to override the onclick listener of the anchor

Reply:
Weird i'm testing on firefox and does not work, on chrome works.

Reply:
Ok let me know if you find something !

Reply:
ok i think i have something, i create a function that overrides the onclick behavior of anything that is inside the version div, with this is not necessary to put the `load_readme` everywhere to make it change, the problem is that firefox have a problem using href and onclick.
https://stackoverflow.com/questions/29173813/onclick-and-href-conflict-in-firefox

Reply:
quick update, on firefox seems like a big problem to solve this, i did make a function that removes all the href of the version when the page loads, solving that conflict, since the pip make a post to request this it doesn't trigger that removal and the href links are present, i just make the test and in the browser the links are removed successfully and pip can find the href links needed.

on browser
<img width="669" alt="image" src="https://github.com/pabnas/github-hosted-pypi/assets/39195909/41a5d681-f436-4dd6-8d9e-1bc26ad36821">
on fetch
<img width="609" alt="image" src="https://github.com/pabnas/github-hosted-pypi/assets/39195909/7082e501-5b14-4340-9660-aa98af46f679">


This seems to solve the problem, what is your opinion @astariul?

Reply:
Seems awesome ! Can you push to this PR or create another PR ?

Reply:
i will add those in my PR to the main branch of your repo